src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -1108,6 +1118,23 @@ void Pane::_ControlConnectionStateChangedHandler(const winrt::Windows::Foundatio
     }
 }
 
+winrt::fire_and_forget Pane::_playBellSound(winrt::Windows::Foundation::Uri uri)
+{
+    auto weakThis{ shared_from_this() };
+
+    co_await winrt::resume_foreground(_root.Dispatcher());
+    if (auto pane{ weakThis.get() })
+    {
+        if (s_bellPlayer)
+        {
+            auto source{ winrt::Windows::Media::Core::MediaSource::CreateFromUri(uri) };

```

Comment:
I _suspect_ that if you set a nonexistent bell sound (or, perhaps, one causes a crash!) this will blast off into space.

We already have a "ValidateMediaPaths", though . . . hmm

Reply:
nah it just does nothing

Reply:
tested with

```jsonc

                // "%~%\\Downloads\\memes\\honks\\Honk1.mp3"
                // "C:\\Users\\migrie\\Downloads\\memes\\honks\\Invalid.mp3"
                "C:\\Users\\migrie\\Downloads\\memes\\no.mp4"
```

Reply:
_huh_! TIL